src/fastcs_odin/odin_controller.py

```diff
@@ -24,6 +24,10 @@ class OdinController(Controller):
     """A root ``Controller`` for an odin control server."""
 
     API_PREFIX = "api/0.1"
+    MODULE_FRAME_PROCESSOR = "FrameProcessorAdapter"
+    MODULE_FRAME_RECEIVER = "FrameReceiverAdapter"
+    MODULE_META_WRITER = "MetaListenerAdapter"
+    MODULE_EIGER_FAN = "EigerFanAdapter"
```

Comment:
Could these go in `util.py` just to tidy this class up a bit?

Reply:
Added an enum to util.